_data/slider.yml

```diff
@@ -0,0 +1,13 @@
+
+- title: "Free, easy-to-use software for
+developing <span>world healthcare</span>."
```

Comment:
We need some kind of navigation dots on this carousel. Right now ti seems that changes without any consistency.

Reply:
Impossible to do in this release